internal/migrations/observability/api/pipelines_test.go

```diff
@@ -51,12 +51,12 @@ var _ = Describe("#ConvertPipelines", func() {
 			Labels: map[string]string{"foo": "bar"},
 		}, []obs.FilterSpec{
 			{
-				Name:            "filter-labelPipeline-openShiftLabels",
+				Name:            "filter-labelPipeline-" + openshiftLabelsFilterName,
 				Type:            obs.FilterTypeOpenshiftLabels,
 				OpenShiftLabels: map[string]string{"foo": "bar"},
 			},
 		},
-			[]string{"filter-labelPipeline-openShiftLabels"}),
+			[]string{"filter-labelPipeline-" + openshiftLabelsFilterName}),
```

Comment:
Same comment as above...
```suggestion
			[]string{"filter-labelipeline-" + openshiftLabelsFilterName}),
```